go/registry/api/runtime.go

```diff
@@ -262,3 +327,7 @@ type RuntimeGenesis struct {
 	// invalid iff the State is non-empty or StateRoot is an empty hash.
 	StorageReceipt signature.Signature `json:"storage_receipt"`
 }
+
+// TransactionSchedulerParameters is the transaction scheduler parameters.
+type TransactionSchedulerParameters struct {
```

Comment:
Why is this struct still here? You added `TxnSchedulerParameters` so I you can remove this.

Reply:
I still see the struct?